### DISCUSSION TITLE
Heroku deploy failed: migrate found failed migrations in the target database

### DISCUSSION THREAD

**Discussion by hayatshin** [prisma/prisma#14493] [Q&A]
When I deploy heroku, I keep facing this issue.

```
remote: 32 migrations found in prisma/migrations
remote: 
remote: Error: P3009
remote: 
remote: migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
remote: The `20220726065831_kakao_id` migration started at 2022-07-26 07:04:22.257859 UTC failed
so in order to roll back that migrate with this command   npx prisma migrate resolve --rolled-back "20220726065831_kakao_id",
```

I says this migration is not in a failed state.

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "chungchunon", schema "public" at "localhost:5432"
Error: P3012

```

Migration `20220726065831_kakao_id` cannot be rolled back because it is not in a failed state.
I've struggle this for a long time.. please help me.

How can I resolve heroku deploy?

**Comment by austincrim** [comment]
Hey there @hayatshin 👋,

Could you provide me with the output of this query to your database?
```sql
select * from _prisma_migrations;
```

The `_prisma_migrations` table contains lots of useful metadata about your migrations and will help us debug further 👍.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
